com.unity.render-pipelines.universal/Runtime/ShaderUtils.cs

```diff
@@ -53,5 +53,31 @@ public static bool IsLWShader(Shader shader)
         {
             return s_ShaderPaths.Contains(shader.name);
         }
+
+#if UNITY_EDITOR
+        static readonly string[] s_ShaderGUIDs =
+        {
+            "933532a4fcc9baf4fa0491de14d08ed7",
```

Comment:
We should add test automation to validate we don't break these GUIDs for some reason. Maybe we should check all s_ShaderGUIDs return valid paths in here: https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.universal/Tests/Editor/EditorTests.cs#L57-L65

What do you think?

Reply:
I added a test in the latest commit, how do you proceed to run them locally ? I ended up copying that folder in a sandbox project and managed to run them, not ideal..

Reply:
You can run them locally by opening the test runner window and running editor mode tests